bin/tools.py

```diff
@@ -896,6 +876,36 @@ def run_parsed_arguments(args):
         sys.exit(1)
 
 
+def read_personal_config():
+    args = {}
+
+    if is_windows():
+        home_config = Path(os.getenv('AppData'))
+    else:
+        home_config = (
+            Path(os.getenv('XDG_CONFIG_HOME'))
+            if os.getenv('XDG_CONFIG_HOME')
+            else Path(os.getenv('HOME')) / '.config'
```

Comment:
You can probably use `or` here as well.

Do we want to fall back to `~/.bapctools.yaml` as well?

Reply:
Using `or` is difficult here, because if `os.getenv(...) = None`, then `Path(None)` will throw an exception.

I don't think we should use multiple config locations for now, I don't like polluting my home dir with random config files anyway :stuck_out_tongue: